_pages/meetings/fall2020.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: page
+title: 2020 Fall Python in Heliophysics Community Meeting
+permalink: /meetings/fall2020/
+exclude: true
+---
+
+The fall 2020 meeting of the [Python in Heliophysics Community](http://heliopython.org) will be held remotely [**at this Zoom link**](https://cuboulder.zoom.us/j/395871054) (Meeting ID: **395 871 054**). It will be split into seven sessions, spread across four weeks, starting Monday, October 26th, 2020 and ending Monday, November 16th, 2020:
+
+ - **Plenary #1:** Oct 26th 9:00 AM–10:00 AM (MT)
+ - **Tutorials:** Oct 29th 11:00 AM–2:00 PM (MT)
+ - **Plenary #2:** Nov 2nd 9:00 AM–10:00 AM (MT)
+ - **Hackathons:** Nov 4th 7:00 AM–10:00 AM (MT)
+ - **Plenary #3:** Nov 9th 9:00 AM–10:00 AM (MT)
+ - **Unconferences:** Nov 11th 7:00 AM–10:00 AM (MT)
+ - **Plenary #4:** Nov 16th 9:00 AM–10:00 AM (MT)
+
+See [the full agenda](https://docs.google.com/spreadsheets/d/18lrxi8XsjDZ76MjWZPFOovu1JWrdjvoLZmyZfS_LZ3A/edit#gid=0) for exact dates/times of the various sessions. 
+
+### Presentations and Documents
+
+Presentations and documents associated will be kept in the [**meeting folder**](https://drive.google.com/drive/u/0/folders/1T3CGRwXAst8jd7I6xFiKxyCgluGGpg0A) on Google Drive.
+
+ - [Meeting schedule](https://docs.google.com/spreadsheets/d/18lrxi8XsjDZ76MjWZPFOovu1JWrdjvoLZmyZfS_LZ3A/edit#gid=0) (subject to change)
+ - [Hackathon projects](https://docs.google.com/spreadsheets/d/1vPgf9nKnJaJ9kYZENxzD83UF84cwi4LQz1_vFLj9Sms/edit#gid=0)
+ - [Presentation, Tutorial, and Unconference Organizing](https://docs.google.com/spreadsheets/d/1dSLU4okGHOlPXGe969C2t9nQWlBr7NOP_F84Hn14WRw/edit#gid=0)
+
+### Session Details?
+
+Could include info about the seven individual sessions here...
```

Comment:
@jibarnum This is what I talked about in the PR description. What do you think?

Reply:
Yeah, the Spring 2020 meeting definitely was more pared down than what we're doing for the Fall 2020 meeting ==> way less information to be communicated on that meeting page than this one. I think it'd be fine to link to the info on Google Drive.

Reply:
Same page then. I’ll wack this Session Details section.

Reply:
Done.